phamos/phamos/doctype/accounting_receipt/accounting_receipt.py

```diff
@@ -34,4 +37,39 @@ def make_purchase_invoice(self):
 		pi.set_missing_values()
 		pi.insert()
 		self.db_set('purchase_invoice', pi.name)
-		return pi
\ No newline at end of file
+		return pi
+
+
+@frappe.whitelist()
+def make_accounting_receipt(issue):
+	"""
+		Create an Accouting Receipt from an Issue
+	"""
+	issue_doc = frappe.get_doc("Issue",issue)
+	ar = frappe.new_doc("Accounting Receipt")
+	ar.posting_date = today()
+	ar.customer = issue_doc.customer
+	ar.project = issue_doc.project
+	ar.title = issue_doc.subject
+	ar.conversion_rate = 1
+	ar.insert()
+	issue_doc.db_set("accounting_receipt", ar.name)
+
+	""" Get the email data and attachments """
+	communication_data = get_communications("Issue",issue)
+	for data in communication_data:
+		content = data.content
+		attachments = frappe.get_all(
+		"File",
+		fields=["name"],
+		filters={"attached_to_name": "Communication", "attached_to_doctype": data.name},
+		)
+		""" Transfer the attachments """
```

Comment:
Not sure about this part requirement, but i think we should create new File and link it with Accounting Receipt instead of changing reference of existing File (attached_to_doctype, attached_to_name)

Reply:
In my opinion changing is fine. We have no value of duplicating the data here.